### PR TITLE
update to allow inclusion of random num generator for TSVD

### PR DESCRIPTION
**Pull request by pazzo83**
Allow usage of random number generator that gets passed to underlying TSVD method

### CODE REVIEW

**Review by ablaom — APPROVED** [review]
This looks great! Suggestions:

- For consistency with other MLJ interfaces, you could allow the option that user provides an integer for `rng` which becomes the seed of a `MersenneTwister`. 

- I would make the fallback value `Random.GLOBAL_RNG` (what TSVD.jl is implicitly using, as I understand it) for otherwise the implementation is technically breaking.

File: src/MLJTSVDInterface.jl [hunk 1]
```diff
@@ -19,15 +19,12 @@ decomposition (SVD). Contrary to PCA, this estimator does not center the data be
...
 
 """
-mutable struct TSVDTransformer{R <: AbstractRNG} <: MLJModelInterface.Unsupervised
-    nvals::Int
-    maxiter::Int
-    rng::R
+MMI.@mlj_model mutable struct TSVDTransformer <: MLJModelInterface.Unsupervised
+    nvals::Int = 2
+    maxiter::Int = 1000
+    rng::Int = 123
```
Comment (ablaom) [inline src/MLJTSVDInterface.jl:25]: No, sorry, I should have been more explicit. I had rather in mind `Union{Integer,AbstractRNG}`. See for example [here](rng::Union{Int,AbstractRNG}) for an example of typing, and [here](https://github.com/JuliaAI/MLJEnsembles.jl/blob/7ca66d70a67cc7d9844e4a1b3365bf5b449b5cc0/src/ensembles.jl#L463) for an example of pre-processing in `fit`.

Comment (pazzo83) [reply src/MLJTSVDInterface.jl]: no worries - the most recent change should reflect this!

**Review by ablaom — APPROVED** [review]
Looks good to me. Thanks for accomodating those suggestions.

**Pull request merged**